test/cppsuite/test_harness/random_generator.h

```diff
@@ -12,22 +11,22 @@ namespace test_harness {
 /* Helper class to generate random values. */
 class random_generator {
     public:
-    static random_generator *
+    /* No copies of the singleton allowed. */
+    random_generator(random_generator const &) = delete;
+    void operator=(random_generator const &) = delete;
+
+    static random_generator &
     get_instance()
     {
-        if (!_instance)
-            _instance = new random_generator;
-        return (_instance);
+        static random_generator _instance;
+        return _instance;
     }
 
     std::string
     generate_string(std::size_t length)
     {
         std::string random_string;
```

Comment:
This can be a const function as well.

Reply:
This function cannot be const as per line 31